packages/menu-bar/src/vaadin-menu-bar-buttons-mixin.js

```diff
@@ -4,13 +4,14 @@
  * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
  */
 import { ResizeMixin } from '@vaadin/component-base/src/resize-mixin.js';
+import { InteractionsMixin } from './vaadin-menu-bar-interactions-mixin.js';
```

Comment:
What are these `InteractionsMixin` related changes?

Reply:
It is because now we have `_themeChanged` in both `InteractionsMixin` and `ButtonsMixin`.

1. One option was to do something like this:
```js
_themeChanged(theme) {
  if (super._themeChanged) {
    super._themeChanged(theme);
  }
}
```

2. Another option was to make `ButtonsMixin` inherit `InteractionsMixin` and override the `_themeChanged` method.

```js
_themeChanged(theme) {
  super._themeChanged(theme);
}
```

It is probably not the best choice to inherit the class just for the sake of one method. Anyway, would you suggest using (1)? Or do you have any other ideas on how it could be done better?

Reply:
The logic related to `theme` property could be moved to `vaadin-menu-bar.js` to make things easier.
Alternatively, we could merge both mixins, there is not much sense in keeping them separate.

Reply:
Pushed an improvement based on what @web-padawan suggested.